test/e2e/space_test.go

```diff
@@ -121,18 +120,138 @@ func TestCreateSpace(t *testing.T) {
 	})
 }
 
-func TestUpdateSpace(t *testing.T) {
+func TestSpaceRoles(t *testing.T) {
+
+	// make sure everything is ready before running the actual tests
+	awaitilities := WaitForDeployments(t)
+	hostAwait := awaitilities.Host()
+	memberAwait := awaitilities.Member1()
 
 	// given
+	s := NewSpace(hostAwait.Namespace, GenerateName("oddity"), "appstudio", WithTargetCluster(memberAwait.ClusterName))
+
+	// when
+	err := hostAwait.CreateWithCleanup(context.TODO(), s)
+
+	// then
+	require.NoError(t, err)
+	// wait until NSTemplateSet has been created and Space is in `Ready` status
+	s, err = hostAwait.WaitForSpace(s.Name,
+		wait.UntilSpaceHasConditions(Provisioned()),
+		wait.UntilSpaceHasStatusTargetCluster(memberAwait.ClusterName),
+	)
+	require.NoError(t, err)
+	// wait until NSTemplateSet has been created and Space is in `Ready` status
+	nsTmplSet, err := memberAwait.WaitForNSTmplSet(s.Name, wait.UntilNSTemplateSetHasConditions(Provisioned()))
+	require.NoError(t, err)
+	tierChecks, err := tiers.NewChecks("appstudio")
+	require.NoError(t, err)
+	tiers.VerifyGivenNsTemplateSet(t, memberAwait, nsTmplSet, tierChecks, tierChecks, tierChecks.GetExpectedTemplateRefs(hostAwait))
```

Comment:
you could use the function `VerifyResourcesProvisionedForSpaceWithTier`
```suggestion
	// wait until NSTemplateSet has been created and Space is in `Ready` status
	VerifyResourcesProvisionedForSpaceWithTier(t, awaitilities, memberAwait, s.Name, "appstudio")
```